client/state/data-layer/wpcom-api-middleware.js

```diff
@@ -17,16 +17,16 @@ const mergedHandlers = mergeHandlers(
 const shouldNext = action => {
 	const meta = action.meta;
 	if ( ! meta ) {
-		return true;
+		return false;
 	}
 
 	const data = meta.dataLayer;
 	if ( ! data ) {
-		return true;
+		return false;
```

Comment:
Why is this changed to `false` here and above? If they have no meta don't we definitely want to pass them along the middleware chain? Only _if_ it has meta _and_ the identified network meta do we not pass it along…